website/docs/examples/camera_calibration/README.md

```diff
@@ -5,9 +5,11 @@ title: Camera Calibration
 
 ## Camera Calibration
 
-The requirements to run this example are to have a
-[**farm-ng brain**](/docs/brain/) running Oak cameras and that
-your PC is on the same local network as the brain.
+The requirement to run this example is to have a
+[**farm-ng brain**](/docs/brain/) running Oak cameras. This can be run either
+directly from a brain or from your local PC. If running from your local PC,
+ensure it's connected to the same local network as the brain
+or linked to it through tailscale.
```

Comment:
To keep things more human readable in the markdown, always break lines on periods and try to break other lines near punctuation, when possible.

```suggestion
The requirement to run this example is to have a
[**farm-ng brain**](/docs/brain/) running Oak cameras.
This can be run either directly from a brain or from your local PC.
If running from your local PC,
ensure it's connected to the same local network as the brain or linked to it through tailscale.
```